README.md

```diff
@@ -21,9 +21,9 @@ And then to the module level `build.gradle.kts`:
 
 ```
 dependencies {
-    implementation 'com.revolut.rxdata:dod:1.5.14'
-    implementation 'com.revolut.rxdata:core:1.5.14'
-    implementation 'com.revolut.rxdata:scheduler:1.5.14'
+    implementation 'com.revolut.rxdata:dod:1.5.15'
```

Comment:
Super optional: 

There might a release version badge be generated through https://shields.io/badges/sonatype-nexus-releases
Example:
![Sonatype Nexus (Releases)](https://img.shields.io/nexus/r/com.revolut.rxdata/dod?server=https%3A%2F%2Fs01.oss.sonatype.org)

Maybe versions could be seen from there rather than fixing it here. Again, super optional and not even minor thing